std_msgs/Header header

float64 lat      # latitude (deg)
float64 lon      # longitude (deg)

float64 x        # x coordinate (m)
float64 y        # y coordinate (m)
float64 psi      # yaw angle (rad)
float64 v        # speed (m/s)

float64 v_long   # longitidunal velocity (m/s)
float64 v_lat    # lateral velocity (m/s)
float64 yaw_rate # w_z, yaw rate (rad/s)

float64 a_long   # longitudinal acceleration (m/s^2)
float64 a_lat    # lateral acceleration (m/s^2)
float64 df       # front steering angle (rad)